.gitignore

```diff
@@ -4,7 +4,11 @@ cxx/build/
 docs/build/
 docs/source/doxygen
 docs/source/mspass_schema/*.csv
-**/_build/
```

Comment:
Should never remove entries in .gitignore

Reply:
Got it.

Reply:
Actually, let's revert the change to this file since you are able to build the package with pip now.

Reply:
Yes! The only things I add here are
**/build/(the folder created in my project after build)
**/mspasspy.egg-info(same above)
.coverage*(files created after running pytest)

Are those path are good to add to .gitignore?

Reply:
It is fine to add those, but you need to remove the `*` because it will match any thing with that name under any sub-directories, which is not good. Note that I had things like `**/_build/` mostly because they start with `_`, which is distinctively different from normal files.

Also, if you build with the pip command, `build` and `mspasspy.egg-info` shouldn't be created under current directory.